src/utils/testing/setup.ts

```diff
@@ -5,3 +5,22 @@ jest.mock('next/font/google', () => ({
     },
   }),
 }));
+
+jest.mock('react-dnd-html5-backend', () => ({
```

Comment:
you found the thing! :D

Reply:
Somehow the drag-and-drop library gets into the dependencies of the unit test. I'd like to investigate at some point why this is happening exactly and find a better remedy. But for now this also worked :)